src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -677,11 +677,11 @@ protected override void OnNodePropertyChanged(object sender, PropertyChangedEven
 
         public override void GenerateViewGeometryFromRenderPackagesAndRequestUpdate(IEnumerable<IRenderPackage> taskPackages)
         {
-            foreach (var p in taskPackages)
+            /*foreach (var p in taskPackages)
             {
                 Debug.WriteLine(string.Format("Processing render packages for {0}", p.Description));
             }
-
+            */
```

Comment:
Correct... but this makes Dynamo unusable during debugging for models with large geometry sets in the view (ie 10,000 points).  If the developer needs the info, they can put it back in